ParabolInc/parabol
# feat: add an archive checkbox in timeline history page

# Description
- **feat: add archive checkbox in timeline history**
- **fix: fix the bug where user cannot archive poker or team prompt meetings**
- **fix: fix an issue where archived timeline event history did not get removed immediately from the view**
- **fix: fix the issue where newly ended meeting won't show up on timeline event list**


## Demo
![2024-10-31 11 47 56](https://github.com/user-attachments/assets/8c3e88fc-9869-4fc2-9907-1744a159a9a3)

## Testing scenarios

- [ ] Toggle archive checkbox in timeline history
  - Go to `http://localhost:3000/me`
  - Toggle the archive checkbox to display/hide archived timeline history

- [ ] Archive a meeting card
  - Click the 3 dot menu of a completed meeting card
  - Archive the meeting
  - See the card disappears
  - Check the archive checkbox to see the newly archived meeting card

## Final checklist
